### ISSUE TITLE
Missing WCAG § numbers 5.2.4 and 5.2.5

### ISSUE THREAD

**Issue by LaurenceZaysser** [w3c/epub-specs#2887]
### Section

https://www.w3.org/TR/epub-a11y-eaa-mapping/#annex-I-section-IV-f-ii

### Describe the problem

Missing WCAG § numbers on two WCAG criterion
WCAG conformance requirement [[5.2.5] Non-Interference](https://www.w3.org/TR/WCAG2/#cc4)
WCAG conformance requirement [[5.2.4] Only Accessibility-Supported Ways of Using Technologies](https://www.w3.org/TR/WCAG2/#cc5)

### Describe the fix or new feature you propose

Add missing paragraph numbers before paragraph names

WCAG conformance requirement [5.2.5 Non-Interference](https://www.w3.org/TR/WCAG2/#cc4)
WCAG conformance requirement [5.2.4 Only Accessibility-Supported Ways of Using Technologies](https://www.w3.org/TR/WCAG2/#cc5)
